### ISSUE TITLE
Server?

### ISSUE THREAD

**Issue by CatsAreFluffy** [inkron/Server#3]
Is this dead? This seems dead. Also, the AT topic is no longer viewable :/

**Comment by encloinc** [comment]
I did some work on it after the team fell apart but yeah its dead rn. That
doesn't mean we can't resurrect it though :)

**Comment by CatsAreFluffy** [comment]
Yeah, I've done some stuff too. Have you uploaded your changes to Github?

**Comment by encloinc** [comment]
No they are all local
